### PR TITLE
UI design

### PR DESCRIPTION
**Pull request by spanickroon**

### CODE REVIEW

**Comment by spanickroon** [comment]
https://marketplace-service.herokuapp.com/login

File: static/js/marketplace/main.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    var input = document.getElementById('change-password-input');
+    showHidePassword(target, input);
+    return false;
+}
+
+function showHidePassword5(target) {
+    var input = document.getElementById('change-password-input-repeat');
+    showHidePassword(target, input);
+    return false;
+}
```
Comment (gmltA) [inline static/js/marketplace/main.js]: `showHidePassword4`
`showHidePassword5`

Сіквелы, якія мы не заслугоўваем!

File: templates/authentication/base.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{% load static %}
+<div class=container-page-authentication>
+    <div class="logo-container">
+        <img class="img-logo" src="{% static 'img/logo.svg' %}" alt="">
+        <label class="label-logo" for="...">PLOTUS STORE</label>
```
Comment (gmltA) [inline templates/authentication/base.html]: `label` doesn't belong here.

File: templates/authentication/login.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            <g clip-path="url(#clip0)">
+            <path d="M18.7259 10.5H17.6023V7.125C17.6023 3.19687 14.4096 0 10.4865 0C6.56342 0 3.37066 3.19687 3.37066 7.125V10.5H2.2471C1.00652 10.5 0 11.5078 0 12.75V21.75C0 22.9922 1.00652 24 2.2471 24H18.7259C19.9665 24 20.973 22.9922 20.973 21.75V12.75C20.973 11.5078 19.9665 10.5 18.7259 10.5ZM13.8571 10.5H7.11583V7.125C7.11583 5.26406 8.62794 3.75 10.4865 3.75C12.345 3.75 13.8571 5.26406 13.8571 7.125V10.5Z"/>
+            </g>
+            <defs>
+            <clipPath id="clip0">
+            <rect width="20.973" height="24" fill="white"/>
+            </clipPath>
+            </defs>
+        </svg>
+        <label class="authentication-label" for="...">Password</label>
```
Comment (gmltA) [inline templates/authentication/login.html]: `for` attribute is incorrect.

File: templates/authentication/login.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        <svg class="password-eye" onclick="return showHidePassword1(this);" aria-hidden="true" focusable="false" data-prefix="far" data-icon="eye" class="svg-inline--fa fa-eye fa-w-18" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 576 512"><path d="M288 144a110.94 110.94 0 0 0-31.24 5 55.4 55.4 0 0 1 7.24 27 56 56 0 0 1-56 56 55.4 55.4 0 0 1-27-7.24A111.71 111.71 0 1 0 288 144zm284.52 97.4C518.29 135.59 410.93 64 288 64S57.68 135.64 3.48 241.41a32.35 32.35 0 0 0 0 29.19C57.71 376.41 165.07 448 288 448s230.32-71.64 284.52-177.41a32.35 32.35 0 0 0 0-29.19zM288 400c-98.65 0-189.09-55-237.93-144C98.91 167 189.34 112 288 112s189.09 55 237.93 144C477.1 345 386.66 400 288 400z"></path></svg>
+    </div>
+    <div id="invalid-popup" class="invalid-popup"></div>
+</form>
+<a class="authentication-nav nav-target" data-target="signup"  href="#">
```
Comment (gmltA) [inline templates/authentication/login.html]: `href` is incorrect.

File: templates/marketplace/base.html [hunk 5]
```diff
@@ -0,0 +1,48 @@
+{% load static %}
+<header class="dashboard">
+    <a href="#" class="nav-target" data-target="profile">username, 14000$</a>
+</header>
+<nav class="nav-bar">
+    <div class="nav-links">
+        <a draggable="false" href="#" class="nav-target" data-target="stocks">
```
Comment (gmltA) [inline templates/marketplace/base.html:7]: Draggable?

**Review by gmltA — CHANGES_REQUESTED** [review]

File: static/js/marketplace/main.js (hunk 1 above)
Comment (spanickroon) [reply static/js/marketplace/main.js]: Поправил

File: templates/authentication/base.html (hunk 2 above)
Comment (spanickroon) [reply templates/authentication/base.html]: Поправил

File: templates/authentication/login.html (hunk 3 above)
Comment (spanickroon) [reply templates/authentication/login.html]: Поправил

File: templates/authentication/login.html (hunk 4 above)
Comment (spanickroon) [reply templates/authentication/login.html]: Поправил

File: templates/marketplace/base.html (hunk 5 above)
Comment (spanickroon) [reply templates/marketplace/base.html:7]: )

**Review by gmltA — APPROVED** [review]

**Pull request merged**